src/plugins/telemetry/server/config/config.ts

```diff
@@ -33,6 +33,13 @@ const configSchema = schema.object({
   sendUsageFrom: schema.oneOf([schema.literal('server'), schema.literal('browser')], {
     defaultValue: 'server',
   }),
+  appendServerlessChannelsSuffix: schema.conditional(
+    schema.contextRef('serverless'),
+    true,
+    schema.literal(true),
+    schema.literal(false),
+    { defaultValue: schema.contextRef('serverless') }
+  ),
```

Comment:
NIT: I wonder if defining a `channelSuffix` config option would have been more generic / re-usable. But also, I tried to set the `defaultValue` in that case and I couldn't really find how...

```ts
  channelSuffix: schema.conditional(
    schema.contextRef('serverless'),
    true,
    schema.literal('serverless'),
    schema.literal(undefined),
    { defaultValue: ??? }
  ),
```

I wonder if this will be an issue in other serverless config madness use cases....